### PR TITLE
updated azure sentinel template 

### LINKED ISSUE #312: Error while creating build definition: The pipeline is not valid.
Error while creating build definition: The pipeline is not valid. A task is missing. The pipeline references a task called '584bcff2-3353-4f11-872b-6ba01267a972'. This usually indicates the task isn't installed, and you may be able to install it from the Marketplace: https://marketplace.visualstudio.com. (Task version 5.*, job 'Phase_1', step ''.) Error while creating build definition: The pipeline is not valid. A task is missing. The pipeline references a task called '584bcff2-3353-4f11-872b-6ba01267a972'. This usually indicates the task isn't installed, and you may be able to install it from the Marketplace: https://marketplace.visualstudio.com. (Task version 5.*, job 'Phase_1', step ''.)

### LINKED ISSUE #313: ARM Outputs:5
Error while creating release definition: Tasks with versions 'ARM Outputs:5.*' are not valid for deploy job 'Agent job' in stage Stage 1.

when I create project ,meet the error.

### LINKED ISSUE #314: Error while creating release definition: Tasks with versions 'ARM Outputs:5.*' are not valid for deploy job 'Agent job' in stage Stage 1.

### PR DESCRIPTION
**Pull request by aksh-h**
[AB#3458](https://vstsdemodata.visualstudio.com/89b9b739-9a1c-4739-8aae-6a8a1eb67ee7/_workitems/edit/3458)

[AB#3460](https://vstsdemodata.visualstudio.com/89b9b739-9a1c-4739-8aae-6a8a1eb67ee7/_workitems/edit/3460)

This resolves #312 resolves #313 resolves #314

### CODE REVIEW

File: src/VstsDemoBuilder/Templates/AzureSentinel/ServiceEndpoints/Azure Sentinel Template-code.json [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
+  "data": {},
   "name": "Azure Sentinel Template-code",
   "type": "git",
   "url": "https://dev.azure.com/vandyr/Azure Sentinel Template/_git/Azure Sentinel Template",
   "authorization": {
     "scheme": "UsernamePassword",
     "parameters": {
       "username": "$username$",
-      "password": "$password$"
+      "password": "$username$"
```
Comment (dmckinstry) [inline src/VstsDemoBuilder/Templates/AzureSentinel/ServiceEndpoints/Azure Sentinel Template-code.json:10]: Is this correct?  Using $username$ for the password?

File: src/VstsDemoBuilder/Templates/TemplateSetting.json [hunk 2]
```diff
@@ -30,7 +30,8 @@
           ],
           "Author": "Microsoft",
           "Image": "/Templates/TemplateImages/TailwindTraders.png",
-          "ForkGitHubRepo": true
+          "ForkGitHubRepo": true,
+          "Message": "Note: This template erquires latest versions of ARM Outputs extension, please update. Ignore if you already have the latest version."
```
Comment (dmckinstry) [inline src/VstsDemoBuilder/Templates/TemplateSetting.json:34]: Typo - "requires" <-- "erquires"

File: src/VstsDemoBuilder/Templates/TemplateSetting.json [hunk 3]
```diff
@@ -50,7 +51,8 @@
             "/Templates/SmartHotel360/Images/Screens.png"
           ],
           "Author": "Microsoft",
-          "ForkGitHubRepo": true
+          "ForkGitHubRepo": true,
+          "Message": "Note: This template erquires latest versions of ARM Outputs extension, please update. Ignore if you already have the latest version."
```
Comment (dmckinstry) [inline src/VstsDemoBuilder/Templates/TemplateSetting.json:55]: Same type

File: src/VstsDemoBuilder/Templates/AzureSentinel/ServiceEndpoints/Azure Sentinel Template-code.json (hunk 1 above)
Comment (aksh-h) [reply src/VstsDemoBuilder/Templates/AzureSentinel/ServiceEndpoints/Azure Sentinel Template-code.json:10]: I'm sorry, didn't notice while copying 🙂

Comment (Damovisa) [reply src/VstsDemoBuilder/Templates/AzureSentinel/ServiceEndpoints/Azure Sentinel Template-code.json]: Still looks like this says `$username$`!

**Comment by christophe-havard-sonarsource** [comment]
@aksh-h does this fix have been deployed yet?

**Comment by aksh-h** [comment]
@christophe-havard-sonarsource ARM Output fix is already there in production and we have a note as well to update the extension
image

Please let me know which template you have tried.

**Comment by christophe-havard-sonarsource** [comment]
@aksh-h I tried the SmartHotel360 template. I don't understand where you put the note you mention. I have the latest one installed on my side.
Just tried the MyHealthClinic template and it works fine ;)

**Comment by aksh-h** [comment]
@christophe-havard-sonarsource I believe you are trying in production [instance](https://azuredevopsdemogenerator.azurewebsites.net/)

If you are using SmartHotel360 template, you should be able to find message as bellow

image

Let me know if anything is not working.

**Comment by christophe-havard-sonarsource** [comment]
yes, as I said, I already have the latest, but it still doesn't work.
However, I just noticed that I don't have the note you mention, which can be explained by the fact that I don't use the same URL as you. Which one is the "right" one?
image

**Comment by aksh-h** [comment]
[Here ](https://demogen-code.azurewebsites.net/)we are testing the **Asp.Net core - Preview** version, which might be missing few latest changes. You can use [production instance](https://azuredevopsdemogenerator.azurewebsites.net/)

**Comment by christophe-havard-sonarsource** [comment]
@aksh-h  acknowledged, thank you for your support ;)

**Pull request merged**